plc4j/api/src/main/java/org/apache/plc4x/java/api/messages/items/ReadRequestItem.java

```diff
@@ -52,6 +64,8 @@ public boolean equals(Object o) {
             return false;
         }
         ReadRequestItem<?> that = (ReadRequestItem<?>) o;
+        // TODO 01.18.18 jf: we should also call the comparison of super at least otherwise this can lead to unwanted behavior.
```

Comment:
Aren't we doing exactly this super check 4 lines above?

Reply:
I'm sorry, i really completely missed that... you're right.